packages/design-system/src/components/Dropdown/Dropdown.stories.tsx

```diff
@@ -26,7 +26,7 @@ type Story = StoryObj<typeof Dropdown>;
 
 const dropdownOptions = [
   { label: '- Select an option -', value: '' },
-  { label: 'Confederated Tribes and Bands of the Yakama Nation', value: '1' },
+  { label: 'Confederated Tribes and Bands of the Yakama Nation', value: '1', key: 'foo' },
```

Comment:
Why is this needed?

Reply:
er... i was just testing your PR reviewing skills lol (thanks for catching this)